waku/waku_api/rest/doc/filterapi_v1_messages.yaml

```diff
@@ -0,0 +1,41 @@
+# Legacy support for v1 waku filter
+# /filter/v1/messages/{contentTopic}:
+get: # get_waku_v2_filter_v1_messages
+  summary: Get the latest messages on the polled content topic
+  description: Get a list of messages that were received on a subscribed content topic after the last time this method was called.
+  operationId: getMessagesByTopic
+  tags:
+    - filter_legacy
+  parameters:
+    - in: path
+      name: contentTopic  # Note the name is the same as in the path
+      required: true
+      schema:
+        type: string
+      description: Content topic of message
+  responses:
+    '200':
+      description: The latest messages on the polled topic.
+      content:
+        application/json:
+          schema:
+            $ref: './schemas/apitypes.yaml#/FilterGetMessagesResponse'
+    # TODO: Review the possible errors of this endpoint
+    '400':
+      description: Bad request.
+      content:
+        text/plain:
+          schema:
+            type: string
+    '404':
```

Comment:
Hm, I need to test it, although I think we should check if we have subscription for the given contentTopic only. 
Than its a bad request.
This can be an issue for implementation rather. But here I will give a better explanation!